README.md

```diff
@@ -90,33 +62,50 @@ After you finish this process, you should be able
 to type `python --version` and get `Python 3.7.2` or other version after
 `Python 3.6.8` in your terminal.
 
-## Initial Setup
+As a Python 3 program, GatorGrouper relies on
+[Pipenv](https://github.com/pypa/pipenv) for the installation of the libraries
+on which it depends and the creation of the virtual environments in which it
+runs. To install GatorGrader, you should first follow Pipenv's installation
+instructions. You should also ensure that you have installed Git on your
+computer and that you can run Git commands in a terminal window. Then, you can
+type the following command in your terminal window to clone GatorGrader's GitHub
+repository:
+
+```
+git clone git@github.com:GatorEducator/gatorgrouper.git
+```
 
-Ensure that you have installed gspread and oauth2client in the root
-directory of the repository. In the terminal use the command:
+If you plan to develop new features for GatorGrader or if you want to run the
+tool's test suite in [Pytest](https://github.com/pytest-dev/pytest), then you
+will need to install the developer dependencies by typing `pipenv install --dev`
+in the directory that was cloned. If you only want to use GatorGrouper,
+then you can type `pipenv install` instead. Once these commands complete
+successfully, that's all you you have to do to install GatorGrouper!
 
-```shell
-python3 -m pip install --user gspread oauth2client
+## Testing GatorGrouper
+
+GatorGrader uses [Pytest](https://docs.pytest.org/en/latest/) for testing.
+Depending on your goals, there are several different configurations in which you
+can run the provided test suite. If you want to run the test suite to see if all
+of the test cases are correctly passing, then you can type one of the following
```

Comment:
This says "one of the following" but there is only one example following.

Reply:
This should be resolved.